python/ingestion/dataset_utils.py

```diff
@@ -1,10 +1,11 @@
 import pandas as pd  # type: ignore
+import numpy as np  # type: ignore
 from ingestion import gcs_to_bq_util
 import ingestion.standardized_columns as std_col
 import ingestion.constants as constants
 
 
-def generate_pct_share_col(df, raw_count_col, pct_share_col, breakdown_col, all_val):
+def generate_pct_share_col(df, raw_count_to_pct_share, breakdown_col, all_val):
```

Comment:
Especially since this fn signature is changing, we should fill in this doc string with explanation of how to send in the `{raw: pct_share}` object

Reply:
Right! Good catch